services/web-app/data/libraries.js

```diff
@@ -388,41 +388,46 @@ const devLibraries = {
   },
   'ibmdotcom-react': {
     host: 'github.com',
-    org: 'mattrosno',
+    org: 'francinelucca',
```

Comment:
Once https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8500 is merged, just a reminder to update the `org`s here to `carbon-design-system`, remove the `ref`, and update `prodLibraries` too.